meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx6ullevk/boot.cmd

```diff
@@ -6,22 +6,9 @@ echo "Using ${fdt_file}"
 setenv bootlimit 3
 setenv devtype mmc
 setenv devnum ${mmcdev}
+setenv rootpart 2
 
 setenv fdt_addr 85800000
 setenv optee_ovl_addr 0x86000000
 
-setenv bootcmd_resetvars 'setenv kernel_image; setenv bootargs; setenv kernel_image2; setenv bootargs2'
-setenv bootcmd_otenv 'run bootcmd_resetvars; ext4load ${devtype} ${devnum}:2 ${loadaddr} /boot/loader/uEnv.txt; env import -t ${loadaddr} ${filesize}'
-setenv bootcmd_load_f 'ext4load ${devtype} ${devnum}:2 ${loadaddr} "/boot"${kernel_image}'
-setenv bootcmd_tee_ovy 'imxtract ${loadaddr}#conf@@FIT_NODE_SEPARATOR@@${fdt_file} fdt@@FIT_NODE_SEPARATOR@@${fdt_file} ${fdt_addr}; fdt addr ${fdt_addr}; fdt resize 0x1000; fdt apply ${optee_ovl_addr}'
```

Comment:
Apalis is using fdt_file_r instead of fdt_file, so we might have to cross check if this works there as well.

Reply:
strange, I have compared both files with diff and that part was common